example/multi-jvm-example/scripts/startRedirector.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+java -cp example/multi-jvm-example/multi-jvm-example-rest-redirector/target/multi-jvm-example-rest-redirector-*-shaded.jar \
+                -Dpalisade.rest.basePath="http://localhost:8080/alter/this/path" \
+                -Dpalisade.rest.config.path="example/multi-jvm-example/multi-jvm-example-docker-services/configRest.json" \
```

Comment:
For the localhost multi-jvm example I would rather it not be reliant on any of the docker services module, given that module will be dependent on multi-jvm example services modules